querydsl-codegen/src/main/java/com/querydsl/codegen/EntitySerializer.java

```diff
@@ -172,7 +172,7 @@ protected void constructorsForVariables(CodeWriter writer, EntityType model) thr
         if (stringOrBoolean) {
             writer.line(thisOrSuper,"(forVariable(variable)",additionalParams,");");
         } else {
-            writer.line(thisOrSuper,"(", localName.equals(genericName) ? EMPTY : "(Class)",
+            writer.line(thisOrSuper,"(", localName.equals(genericName) ? EMPTY : "(Class) ",
```

Comment:
The conflicts are on this line and line 77, they are already resolved.
Consider removing them, since they are not part of name resolving.